File: tasks/translation/output/Qwen/Qwen2.5-Coder-32B-Instruct-AWQ/codenet/direct/inline/comment/direct/Java/Python/s684030429_commented.py
import sys

class Main:
    def __init__(self):
        self.sc = sys.stdin

    def solve(self):
        a = self.ni()
        b = self.ni()

        print(max(0, a - 2 * b))

    def ni(self):
        return int(self.sc.readline())

# main method
if __name__ == "__main__":
    main_instance = Main()
    main_instance.solve()

# End of Code

File: tasks/translation/output/Qwen/Qwen2.5-32B-Instruct-AWQ/codenet/direct/javadoc/uncomment/direct/Java/Go/s684030429_commented.go
package main

import (
	"bufio"
	"fmt"
	"os"
	"strconv"
)

var sc = bufio.NewScanner(os.Stdin)

func nextInt() int {
	sc.Scan()
	num, _ := strconv.Atoi(sc.Text())
	return num
}

func solve() {
	a := nextInt()
	b := nextInt()

	fmt.Println(max(0, a-2*b))
}

func max(a, b int) int {
	if a > b {
		return a
	}
	return b
}

func main() {
	sc.Split(bufio.ScanWords)
	solve()
}

// End of Code
